src/OrthoTree/morton_codes.h

```diff
@@ -473,7 +471,7 @@ inline morton_code Morton<Dim>::get_step_size(morton_code code) const
     // the min step size is equal to floor(log2(max_depth)) + 1 == sizeof(depth encoding)
     // each level above min depth increases step size by Dim bits
     // so simplified: 1 << (depth_mask_shift + Dim * (max_depth - get_depth(code)))
-    return 1 << (depth_mask_shift + Dim * (max_depth - get_depth(code)));
+    return (int64_t)1 << (depth_mask_shift + Dim * (max_depth - get_depth(code)));
```

Comment:
this should probably be morton_code

Reply:
Agreed. int64_t might even be risky because of sign issues (Although I can't think of a case where it fails)